rest-api/tools/import_organizations.py

```diff
@@ -0,0 +1,156 @@
+"""Imports awardees / HPOs, organizations, and sites into the database.
+
+Usage:
+  tools/import_organizations.sh [--account <USER>@pmi-ops.org --project <PROJECT>] \
+      --awardee_file <AWARDEE CSV> --organization_file <ORGANIZATION CSV> \
+      --site_file <SITE CSV> [--dry_run]
+
+  The CSV files originate from the Google Sheets found here:
+
+  https://docs.google.com/spreadsheets/d/1CcIGRV0Bd6BIz7PeuvrV6QDGDRQkp83CJUkWAl-fG58/edit
+"""
+
+import logging
+
+from dateutil.parser import parse
+from tools.csv_importer import CsvImporter
+from dao.hpo_dao import HPODao
+from dao.organization_dao import OrganizationDao
+from dao.site_dao import SiteDao
+from model.hpo import HPO
+from model.organization import Organization
+from model.site import Site
+from model.site_enums import SiteStatus
+from participant_enums import OrganizationType
+from main_util import get_parser, configure_logging
+
+class HPOImporter(CsvImporter):
+
+  def __init__(self):
+    super(HPOImporter, self).__init__('awardee', HPODao(), 'hpoId', 'name',
+                                      ['Awardee ID', 'Name', 'Type'])
+    self.new_count = 0
+
+  def _entity_from_row(self, row):
+    type_str = row['Type']
+    try:
+      organization_type = OrganizationType(type_str)
+      if organization_type == OrganizationType.UNSET:
+        organization_type = None
+    except TypeError:
+      logging.info('Invalid organization type %s for awardee %s', type_str, row['Awardee ID'])
+      return None
+    return HPO(name=row['Awardee ID'],
+               displayName=row['Name'],
+               organizationType=organization_type)
+
+  def _insert_entity(self, entity, existing_map, session, dry_run):
+    # HPO IDs are not autoincremented by the database; manually set it here.
+    entity.hpoId = len(existing_map) + self.new_count
+    self.new_count += 1
+    super(HPOImporter, self)._insert_entity(entity, existing_map, session, dry_run)
+
+
+class OrganizationImporter(CsvImporter):
+
+  def __init__(self):
+    super(OrganizationImporter, self).__init__('organization', OrganizationDao(),
+                                               'organizationId', 'externalId',
+                                               ['Awardee ID', 'Organization ID', 'Name'])
+    self.hpo_dao = HPODao()
+
+  def _entity_from_row(self, row):
+    hpo = self.hpo_dao.get_by_name(row['Awardee ID'])
+    if hpo is None:
+      logging.info('Invalid awardee ID %s importing organization %s', row['Awardee ID'],
```

Comment:
nit: Most of these seem like they should be logged at at least warning level

Reply:
Done.